src/atcore.h

```diff
@@ -125,6 +137,50 @@ public slots:
      */
     void stop();
 
+    /**
+     * @brief pause an in process print job
+     */
+    void pause();
+
+    /**
+     * @brief resume a paused print job.
+     */
+    void resume();
+
+    /**
+     * @brief Send home command
+     * @param axis: the axis(es) to home (use X Y Z or any combo of)
+     */
+    void home(uchar axis);
+
+    /**
+     * @brief Send home all command
+     */
+    void home();
+
+    /**
+     * @brief Set extruder temperature
+     * @param temp : new temperature
+     * @param extruder : extruder number
+     */
+    void setExtruderTemp(uint temp = 0, uint extruder = 0);
+
+    /**
+     * @brief Set the bed temperature
+     * @param temp : new temperature
+     */
+    void setBedTemp(uint temp = 0);
+
+    /**
+     * @brief setFanSpeed set the fan speed
+     * @param fanNumber: fan number
+     * @param speed: new speed of the fan 0-100
+     */
+    void setFanSpeed(uint speed = 0, uint fanNumber = 0);
+
+    void setAbsolutePosition();
+    void setReletivePosition();
```

Comment:
It's relative pos: http://reprap.org/wiki/G-code#M83:_Set_extruder_to_relative_mode